operate/services/manage.py

```diff
@@ -2066,6 +2078,70 @@ def refill_requirements(  # pylint: disable=too-many-locals
             "allow_start_agent": allow_start_agent,
         }
 
+    def _compute_olas_requirements(  # pylint: disable=too-many-locals
+        self, service_config_id: str, chain: str
+    ) -> int:
+        service = self.load(service_config_id=service_config_id)
+        chain_config = service.chain_configs[chain]
+        ledger_config = chain_config.ledger_config
+        user_params = chain_config.chain_data.user_params
+
+        # TODO fix this
+        os.environ["CUSTOM_CHAIN_RPC"] = ledger_config.rpc
+
+        if not user_params.use_staking:
+            return 0
+
+        # First, compute required OLAS
+        sftxb = self.get_eth_safe_tx_builder(ledger_config=ledger_config)
+        staking_params = sftxb.get_staking_params(
+            staking_contract=STAKING[ledger_config.chain][
+                user_params.staking_program_id
+            ],
+        )
+
+        required_olas = (
+            staking_params["min_staking_deposit"]
+            + staking_params["min_staking_deposit"]  # bond = staking
+        )
```

Comment:
@kupermind is this ok?

Reply:
can be put as:
`2 * staking_params["min_staking_deposit"]`?